vpr/src/pack/prepack.cpp

```diff
@@ -1548,6 +1548,7 @@ static void update_chain_root_pins(t_pack_patterns* chain_pattern,
     for (const auto pin_ptr : chain_input_pins) {
         std::vector<t_pb_graph_pin*> connected_primitive_pins;
         get_all_connected_primitive_pins(pin_ptr, connected_primitive_pins);
+        VTR_ASSERT(connected_primitive_pins.size());
```

Comment:
Suggest a comment here on the assert.  I think the assert is checking that *some* primitive pin in *some* mode connects to the cluster level chain input pin.